sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import uuid
+from base64 import b64decode
+from datetime import datetime
+from azure.core._utils import _convert_to_isoformat, TZ_UTC
+
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
+
+if TYPE_CHECKING:
+    from typing import Any, Dict
+
+
+__all__ = ["CloudEvent"]
+
+
+class CloudEvent(object):  # pylint:disable=too-many-instance-attributes
+    """Properties of the CloudEvent 1.0 Schema.
+    All required parameters must be populated in order to send to Azure.
+    If data is of binary type, data_base64 can be used alternatively. Note that data and data_base64
```

Comment:
I don't quite understand this comment. How do I use data_base64 of a CloudEvent instance?

Reply:
this doesn't apply anymore - removed the line